src/lib.rs

```diff
@@ -0,0 +1,609 @@
+use std::collections::HashMap;
+
+#[cfg(feature = "chunked-entries")]
+mod private;
+#[cfg(feature = "chunked-entries")]
+pub use private::{AbiCombineError, AbiCombineErrorKind, ChunkedAbiEntry};
```

Comment:
If you it this way, aren't these types going to be a part of the public API? Wouldn't it make more sense to do it like this:
```suggestion
#[cfg(feature = "chunked-entries")]
#[doc(hidden)]
#[path = "private.rs"]
pub mod __private;
```

And use everything through `__private`?

Reply:
Also not sure if `chunked-entries` should be a public feature. To me it seems like an implementation detail of how cargo-near interacts with near-sdk.